expense-splitter/src/components/group/CreateGroup.jsx

```diff
@@ -1,71 +1,85 @@
-import { nanoid } from 'nanoid'
-import { useForm } from 'react-hook-form'
-import Button from '../ui/Button'
+import { nanoid } from "nanoid";
+import { useForm } from "react-hook-form";
+import Button from "../ui/Button";
+import { UseDataContext } from "../context/SiteContext";
+import MultiSelectDropdown from "../ui/MultiSelectDropdown";
 
 export default function CreateGroup({
-  groupData,
-  setGroupData,
-  DisplayCreateGroupForm,
+	groupData,
+	setGroupData,
```

Comment:
I'm getting an error here. setGroupData is passed in as props, but also declared on line 12. Does it need to be passed it or is it just pulling in from context?

Reply:
I merged it before seeing this, sorry about that Alex.